templates/corporate/pricing_model.html

```diff
@@ -169,7 +169,7 @@ <h2>Self-managed</h2>
                                 <li><span>Unlimited <a href="/help/start-a-call">voice and video calls</a></span></li>
                                 <li><span>Unlimited <a href="/integrations/">integrations</a></span></li>
                                 <li><span>Easy <a href="https://zulip.readthedocs.io/en/stable/production/install.html">installation</a> and <a href="https://zulip.readthedocs.io/en/stable/production/upgrade.html">maintenance</a></span></li>
-                                <li class="comparison-table-pointer"><span>And <a href="/plans/#self-hosted-plan-comparison">much more</a>!</span></li>
+                                <li class="comparison-table-pointer"><span>And <a href="{{ billing_base_url }}/plans/#self-hosted-plan-comparison">much more</a>!</span></li>
```

Comment:
Could these just be `href="#self-hosted-plan-comparison"`?

Reply:
No, since this is used in other `/self-hosting/` too. Jush pushed another change to provide the correct context on that page too.

Reply:
For some reason chrome doesn't scroll to that section though after clicking on that `much more` link on `self-hosting` page. Looking into fixing that now.